modules/dnn/test/test_layers.cpp

```diff
@@ -1937,6 +1937,171 @@ INSTANTIATE_TEST_CASE_P(/**/, Layer_Test_Eltwise_unequal, Combine(
+        }
+    };
+
+    vector<vector<int>> dim_shape_list;
+    vector<vector<int>> sub_shape_list;
+    bool passed = true;
+    int backend = get<0>(GetParam());
+    int target = get<1>(GetParam());
+
+    string output = format("## BackendID: %d, TargetID: %d\n", backend, target);
+    for (string &op: ops_list)
+    {
+        output += format("### Operator: %s\n", op.c_str());
+        for (int dim: dims_list)
+        {
+            output += format("#### Dimension: %d\n", dim);
+            sub_shape_list.insert(sub_shape_list.end(), dim_shape_list.begin(), dim_shape_list.end());
+            util::get_all_arr(dim_shape_list, dim);
+            util::replace(dim_shape_list, 1, 3);
+            // same shape
+            output += "- **Same Shape**\n";
+            for (int i = 0; i < dim_shape_list.size(); i++)
+                for (int j = 0; j < dim_shape_list.size(); j++)
+                {
+                    int type = util::test_bcast(dim_shape_list[i], dim_shape_list[j], op, backend, target);
+                    util::append_result(output, type, dim_shape_list[i], dim_shape_list[j], op);
+                    if (passed && type != 1)
+                        passed = false;
+                }
+            // diff shape
+            output += "- **Different Shape**\n";
+            for (const auto &shp1: dim_shape_list)
+                for (const auto &shp2: sub_shape_list)
+                {
+                    int type = util::test_bcast(shp1, shp2, op, backend, target);
+                    util::append_result(output, type, shp1, shp2, op);
+                    if (passed && type != 1)
+                        passed = false;
+                }
+
+            // diff shape
+            for (const auto &shp1: sub_shape_list)
+                for (const auto &shp2: dim_shape_list)
+                {
+                    int type = util::test_bcast(shp1, shp2, op, backend, target);
+                    util::append_result(output, type, shp1, shp2, op);
+                    if (passed && type != 1)
+                        passed = false;
+                }
+        }
+    }
+
+    if (passed)
+        cout << "All brute force cases are passed!";
+    else
+        CV_Error(Error::StsInternal, output);
+}
+
+// This test is to verify whether the broadcast operations of unidirectional and bidirectional,
+// as well as tensors with same and different shapes, can be forwarded correctly.
+// This can ensure that the elementwise layer does not have any errors when forwarding.
+// The default is to disable the test, replacing 0 with 1 will enable the test.
+// To test which cases the backend will fallback to the cpu, replace the fallback command like
+// `return Ptr<BackendNode>();` in `initCUDA()` with `throw std::runtime_error("fallback");`
+#if 0
+INSTANTIATE_TEST_CASE_P(/**/, Layer_Test_Eltwise_bcast, dnnBackendsAndTargets());
+#endif
```

Comment:
The best way to disable test in GTest is addtion of `DISABLED_` to it's name. It's listed with disabled status in reports, but could be forced with command line parameters.

Reply:
Thanks, I will change it